File: FlexServ Backend/SpringBoot/AdminService/src/main/java/com/flexserv/dto/response/ServiceResponse.java
package com.flexserv.dto.response;

import java.math.BigDecimal;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class ServiceResponse {

    private Long id;

    private Long categoryId;

    private String categoryName;

    private String name;

    private String description;

    private BigDecimal price;

    private Integer duration;

    private Boolean isActive;
}

File: FlexServ Backend/DotNet/AdminService/Dtos/ServiceResponse.cs
namespace AdminService.Dtos
{
    public class ServiceResponse
    {
        public long Id { get; set; }
        public long? CategoryId { get; set; }
        public string? CategoryName { get; set; }
        public string Name { get; set; } = string.Empty;
        public string? Description { get; set; }
        public decimal? Price { get; set; }
        public int? Duration { get; set; }

        public ServiceResponse() { }

        public ServiceResponse(long id, long? categoryId, string? categoryName, string name, string? description, decimal? price, int? duration)
        {
            Id = id;
            CategoryId = categoryId;
            CategoryName = categoryName;
            Name = name;
            Description = description;
            Price = price;
            Duration = duration;
        }
    }
}
